dist/src/main/resources/application-broker.properties

```diff
@@ -1,5 +1,25 @@
 # Broker specific Spring configuration
 # You should configure broker specific Spring defaults here; things like the actuator endpoints and
 # so on
-# Disable health information for the broker
+# Disable default health information for the broker only to re-enable customized ones
+management.endpoint.health.enabled=true
 management.health.defaults.enabled=false
+management.endpoint.health.cache.time-to-live=1s
+management.endpoint.health.logging.slow-indicator-threshold=10s
+management.endpoint.health.show-details=never
+management.endpoint.health.show-components=never
+management.endpoint.health.probes.enabled=true
+
+# Configure liveness health check
+management.health.livenessstate.enabled=true
+management.endpoint.health.group.liveness.include=brokerLive
+management.endpoint.health.group.liveness.show-details=never
+
+# Configure readiness health check
+management.health.readinessstate.enabled=true
+management.endpoint.health.group.readiness.include=brokerReady
+management.endpoint.health.group.readiness.show-details=never
```

Comment:
:thought_balloon: Would be interesting to provide details in readiness and health check - like which partition is ready/healthy and which is not.

Reply:
Yeah, I thought about that, but then it would be more refactoring (and our health check service only returns boolean :thinking:)

Reply:
Ya. Was just a thought. No need to do it in this PR.